### PR TITLE
Adding Swagger to the API for documentation.

### PR DESCRIPTION
**Pull request by JVanderford**
Added Swagger to the API and included instructions in the readme for …browsing the interactive API documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ These instructions will cover usage information for the API and the optional dev
 After successfully starting the API app, you should be able to view data by browsing to [http://127.0.0.1:3000/api/locations](http://127.0.0.1:3000/api/locations).
 For more in-depth testing, use a web debugging tool such as [Fiddler](https://www.telerik.com/download/fiddler) or [Postman](https://www.getpostman.com/).
 
+### API Documentation
```
Comment (BWortman) [inline README.md]: The README in the Node.js implementation states the following. Could we make this implementation, and README, consistent with that?

### API Route Documentation

Once running locally, you can access API route documentation (useful for those consuming the API) by going to: [http://localhost:3000/docs/api/](http://localhost:3000/docs/api/)

Comment (JVanderford) [inline README.md]: I will look into this.  I just used the basic setup that I saw across multiple guides, my apologies for not looking at the node implementation first.  I will pull down the node app and see how it works there and try to make the aspnetcore implementation consistent with that.

**Comment by JVanderford** [comment]
Implemented requested changes.

**Comment by BWortman** [comment]
Looks good and works. @JeremiahDotNet , approve?

**Pull request merged**